.circleci/config.yml

```diff
@@ -42,6 +42,9 @@ jobs:
       - run:
           name: Setup tailor config
           command: cp ./tailor.config.js.example ./tailor.config.js
+      - run:
+          name: Build core packages
+          command: npm run packages:build
```

Comment:
Can you address the CI build step failure?

Reply:
yes 😫 Sometimes, I get the same issue locally than I rerun `npm install` and everything works...